application/phoenix/README.md

```diff
@@ -0,0 +1,35 @@
+This Phoenix example checks if the environment variables `DEMO_USERNAME` and `DEMO_PASSWORD` have been set. If that's the case, you'll receive a personalized greeting message on http://localhost:4000, otherwise you will get a "Hello stranger" message.
```

Comment:
This "hello stranger" thing is new to me, I'd prefer to change that so it's consistent with the other examples:
 - [Ruby on Rails](https://github.com/secrethub/examples/blob/c8e2b98c03b8a155ed2e0afb620fa71f6afe8b67/application/rails/application_controller.rb)
 - [Django](https://github.com/secrethub/examples/blob/c8e2b98c03b8a155ed2e0afb620fa71f6afe8b67/application/django/views.py)
 - [Flask](https://github.com/secrethub/examples/blob/c8e2b98c03b8a155ed2e0afb620fa71f6afe8b67/application/flask/app.py)

Reply:
After some investigation, this proves to be a little cumbersome to make with my actual skillset in Elixir/Phoenix. I will try to revisit this and to fix it accordingly as soon as I'll have some time to read more on it.

Reply:
This has been fixed. I've tried it out and it now works as expected.